hostresolver/hostresolver_test.go

```diff
@@ -0,0 +1,23 @@
+package hostresolver
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestRules_CNAMEFlatten(t *testing.T) {
+	hostResolver := NewDefaultHostResolver()
```

Comment:
Can you change the test to make it similat to [these tests](https://github.com/containous/traefik/blob/master/provider/acme/provider_test.go)?

We are currently doing a refactoring of our tests in this way.